kernel/virtio/boot.S

```diff
@@ -141,42 +140,58 @@ END(_start)
 
 .data
 .align 64
-.globl cpu_gdt64
-cpu_gdt64:
+gdt64:
 	.quad 0x0000000000000000
-	.quad 0x00af9b000000ffff	/* 64bit CS		*/
+	.quad GDT_DESC_CODE_VAL		/* 64bit CS		*/
```

Comment:
It does not, LIMIT is bits (0 - 15) and (48 - 51) all of which are 1. The values were taken directly from the result of the computation in `ukvm_x86_setup_gdt()`. The reason I made this change was to deliberately remove any confusion/differences between the values used by `ukvm` and those used by `virtio`.